src/components/frame/v5/pages/BalancePage/partials/TokenAvatar/TokenAvatar.tsx

```diff
@@ -52,13 +53,14 @@ const TokenAvatar: FC<TokenAvatarProps> = ({
           <CopyableAddress>{token.tokenAddress}</CopyableAddress>
         )}
         {isTokenNative && !nativeTokenStatus?.unlocked && (
-          <span>
-            <IconTooltip
-              icon="lock"
-              tooltipText={{ id: 'tooltip.lockedToken' }}
-              appearance={{ size: 'tiny' }}
-            />
-          </span>
+          <Tooltip
```

Comment:
sadly this will conflict with [this PR](https://github.com/JoinColony/colonyCDapp/pull/1538), we need to keep in mind to use phosphor icons